ros/src/twist_controller/twist_controller.py

```diff
@@ -52,7 +52,7 @@ def __init__(self,
         # current kp, ki and kd are values found from previous twiddle runs.
         self.steer_pid = PIDWithTwiddle("steering PID", kp=0.607900, ki=0.000172, kd=1.640951,
                                         mn=-max_steer_angle, mx=max_steer_angle,
-                                        optimize_params=False, iterations=10, tolerance=0.05)
+                                        optimize_params=True, iterations=10, tolerance=0.05)
```

Comment:
Victor, do you want to leave optimize_params as True for the next submission?

Reply:
we have two options here:

1. Have a run with `optimize_params=True` on test rosbag and use those parameters for the resubmition ( only if we see an improvement on the test rosbag, .i.e. the car follows waypoints ) and `optimize_params=False`.
2. We simply go ahead and resubmit with `optimize_params=True` and let the algo learn the optimal parameters on the field.

Did you have the chance to test this locally using the test rosbag?